docs/versioned_docs/version-v1.3.0/02-Features/02-message-handling/04-customize-general.md

```diff
@@ -0,0 +1,259 @@
+---
+title: "Customize general message handling"
+layout: default
+---
+
+# Customize general message handling
+
+While the message processing is handled by the `IMessageHandler<>` implementations, the message router controls in what format the message is received.
```

Comment:
What is meant with the format of the message ? Do you mean the content-type of the message ?
And what has the message router to do with it ?  This sentence confuses me a bit tbh

Reply:
The message handler defines the message type it can process, the message router tries to deserialize towards that message type.
I'll change this a bit.

Reply:
I've removed these two sentences as they were already from one of the first versions of messaging.

Reply:
https://github.com/arcus-azure/arcus.messaging/pull/337/commits/693eb3849ca175ac133de7eab21727e5667e51af